final/evaluation.tex

```diff
@@ -65,28 +65,31 @@ \subsection{Evaluation of the Design Goals}
 discussed. Each design goal is listed below, along with an explanation as to how
 it has been met.
 
-\paragraph{Language-agnostic} This design goal stated that no assumptions should
-be made on the host language. The REPL should work for any language, without the
-language designer having to configure it. However, should the language designer
-want additional commands or language-specific configuration of the REPL, this
-should be possible.
-
-Both the frontends and the backend place no assumptions on the language:
-everything automatically works for any language defined in Spoofax. If wanted,
-the language designer can specify REPL-specific configuration in an esv file.
-Currently, however, only DynSem-based interpreters are supported, as opposed to
-Java- and Stratego-based interpreters. Despite this, care has been taken to make
-this easily extendable. This design goal, therefore, is met.
-
-\paragraph{Autogeneration} This design goal stated that the REPL should
-automatically be provided, just like all of Spoofax's other services.
-
-The REPL exists automatically for all languages, due to it reusing the existing
-services offered by Spoofax. This means that it is automatically provided,
-without any effort at the hands of the language designer or -user. Therefore,
-this design goal is met.
+\paragraph{Language-agnostic} This design goal stated that the REPL should work
+for any language. Additionally, the language designer should be provided with a
+language-agnostic interface to configure and implement the language-specific
+parts of the REPL.
+
+An extension of Spoofax's Editor Services has been created, allowing language
+designers to configure certain parts of the REPL. Additionally,
+language-specific changes, such as the ability to redefine identifiers, can be
+made through DynSem. Currently, however, only DynSem-based interpreters are
+supported, as opposed to Java- and Stratego-based interpreters. Despite this,
+care has been taken to make this easily extendable. This design goal, therefore,
+is met.
+
+\paragraph{Minimal configuration} This design goal stated that the REPL should
+require as little configuration as possible by reusing existing components of
+the language definitions and automatically deriving other settings.
+
+The extension of the Editor Services discussed in the previous design goal is
+minimal. Three settings can be configured at the time of writing, all of which
+have sane default settings. One additional setting is mandatory and cannot be
+derived automatically (see \cref{ssec:supp-diff-ways}): the name of the
+evaluation strategy to use. Additionally, the SDF3 configuration is reused to
+specify which start symbols are valid in case the language-designer wants to
+allow different start symbols. Therefore this design goal is met.
```

Comment:
Should this also be discussed in implementation?

Also we should talk about dynsem rules being reusable.